runtime/lua/vim/shared.lua

```diff
@@ -326,6 +326,22 @@ function vim.tbl_islist(t)
   end
 end
 
+-- Return the number of the value in table
+--
+-- This function don't count nil value.
```

Comment:
misleading, is not that it doesn't count it, it just never existed in the first place.

Reply:
I seem to have misunderstood how nil is treated in Lua tables. Thanks for the point.